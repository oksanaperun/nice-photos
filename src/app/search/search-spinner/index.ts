export * from './search-spinner.component';
