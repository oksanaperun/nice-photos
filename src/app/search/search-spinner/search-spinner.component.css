.mat-spinner {
  margin: 30px auto 0 auto;
}
