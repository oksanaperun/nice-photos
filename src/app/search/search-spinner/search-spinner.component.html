<mat-spinner diameter="75"></mat-spinner>
