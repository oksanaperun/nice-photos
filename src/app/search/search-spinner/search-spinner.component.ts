import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-search-spinner',
  templateUrl: './search-spinner.component.html',
  styleUrls: ['./search-spinner.component.css'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})

export class SearchSpinnerComponent {}
