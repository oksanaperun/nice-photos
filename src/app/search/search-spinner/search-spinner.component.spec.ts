import { TestBed, ComponentFixture } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { SearchSpinnerComponent } from './search-spinner.component';

describe('SearchSpinnerComponent', () => {
  let component: SearchSpinnerComponent;
  let fixture: ComponentFixture<SearchSpinnerComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [SearchSpinnerComponent],
      schemas: [NO_ERRORS_SCHEMA],
    }).compileComponents();

    fixture = TestBed.createComponent(SearchSpinnerComponent);
    component = fixture.componentInstance;
  });

  it('should render spinner', () => {
    expect(fixture).toMatchSnapshot();
  });
});
