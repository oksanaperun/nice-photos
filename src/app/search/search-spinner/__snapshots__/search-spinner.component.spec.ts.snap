// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchSpinnerComponent should render spinner 1`] = `
<app-search-spinner>
  <mat-spinner
    diameter="50"
  />
  
</app-search-spinner>
`;
